specification/chaos/resource-manager/Microsoft.Chaos/preview/2023-10-27-preview/types/experiments.json

```diff
@@ -117,6 +122,25 @@
       ],
       "additionalProperties": false
     },
+    "customerDataStorageProperties": {
+      "description": "Model that represents the Customer Managed Storage for an Experiment.",
+      "type": "object",
+      "properties": {
+        "storageAccountResourceId": {
```

Comment:
can this be formatted as arm-id? [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details)

Reply:
Thanks @sjanamma. I've updated with this recommendation.